### ISSUE TITLE
error loading config file <ERs> nonexistent folder: /var/spool/cgrates/cdrc/out for reader with ID: *default

### ISSUE THREAD

**Issue by nunovieira** [cgrates/cgrates#2201]
Just installed `http://pkg.cgrates.org/rpm/v0.10/cgrates_current.rpm` (modified on 27-May-2020 08:42) with no other changes in configuration and now there's an error
```
+ cgr-migrator -config_path=/etc/cgrates '-exec=*set_versions'
2020/06/02 09:57:12 error loading config file <ERs> nonexistent folder: /var/spool/cgrates/cdrc/out for reader with ID: *default
```
The relevant configuration seems to be (trimmed for brevity):
```
  "ers": {
    "enabled": true,
    "sessions_conns": ["*internal"],
    "readers": [
      {
        "id": "file_reader",
        "run_delay": "-1",
        "field_separator": ",",
        "type": "*file_csv",
        "source_path": "/usr/src/cdrs_in",
        "flags": ["*cdrs"],
        "processed_path": "/usr/src/cdrs_out",
        "fields": [
          {"tag": "ToR", "path": "*cgreq.ToR", "type": "*composed", "value": "*voice", "mandatory": true},
// (...)
        ]
      }
    ]
  },
```

**Comment by Trial97** [comment]
Hi @nunovieira,

This issue should be fixed please update to the latest v0.10 rpm package(http://pkg.cgrates.org/rpm/v0.10/cgrates-0.10.2~dev-20200602093155%2B8b81b6b8e.x86_64.rpm) and retest.

Thanks,
Trial97

**Comment by nunovieira** [comment]
When should we expect a v0.10.2?
Please release that soon or make available the rpm of v0.10.0, which doesn't have this issue.

**Comment by danbogos** [comment]
hi  @nunovieira 

Unfortunately we cannot give estimates on when will the next release occur(we have burned ourselves in the past with that). We usually do it when enough issues were applied in order to worth the fuss around announcements. However since we already packaged your fix, I hope there is no big issue in using the package itself on your side.

Thanks,
DanB
